idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/DetectorUtil.java

```diff
@@ -2,13 +2,16 @@
 
 import edu.illinois.cs.dt.tools.runner.data.DependentTest;
 import edu.illinois.cs.dt.tools.runner.data.TestRun;
+import edu.illinois.cs.dt.tools.utility.Level;
+import edu.illinois.cs.dt.tools.utility.Logger;
 import edu.illinois.cs.testrunner.configuration.Configuration;
```

Comment:
I notice that in general we are no longer using the Configuration class, could we just remove the import from all the classes if we are no longer using it?

Reply:
It seems that we still do need this configuration class in 2 particular files, 
idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/SmartShuffler.java and
idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/TestShuffler.java.
We may need to take a closer look.